### PR TITLE
Add German GitOps book

### PR DESCRIPTION
**Pull request by makkes**

### CODE REVIEW

File: RESOURCES.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ The reference material entries listed in this document are contributions coverin
 | *GitOps: Cloud-native Continuous Deployment* |  Anja Kammer, Florian Beetz and Simon Harrer | July 2021 | Innoq Deutschland Gmbh | 9783982112688 |
 | *Repeatability, Reliability, and Scalability through GitOps: Continuous delivery and deployment codified* | Bryan Feuling | May 2021 | Packt | 9781801077798 |
 | *Practical Gitops: Infrastructure Management Using Terraform, Aws, and Github Actions* | Rohit Salecha | December 2022 | Apress | 9781484286722 |
+| [German] *GitOps: Grundlagen und Best Practices* | Baris Cubukcuoglu, Josia Scheitt, Johannes Schnatterer | 2024 | dpunkt.verlag | 9783864909962 |
```
[the remark this answers is not in the document]
Comment (scottrigby) [reply RESOURCES.md:41]: Not bikeshedding, either is fine, just curious if it makes sense or if 99% will be in English anyway. We could revisit once we have enough other languages perhaps?

[the remark this answers is not in the document]
Comment (makkes) [reply RESOURCES.md:41]: Had the same thought. Language column sounds good now that we would be adding the first non-English book.

**Comment by makkes** [comment]
@schnatterer fyi

**Review by scottrigby — APPROVED** [review]
💯 🎉

**Comment by schnatterer** [comment]
I'm honored to see our book on this list. Thanks for adding it and letting me know 🙏

**Pull request merged**